mono/mini/mini-posix.c

```diff
@@ -1036,7 +1034,6 @@ dump_native_stacktrace (const char *signal, void *ctx)
 #endif
 			}
 
-#ifdef TARGET_OSX
```

Comment:
This is the main part, right?

Reply:
It's one of two external entry points into the system